etcdctl/README.md

```diff
@@ -1129,6 +1129,43 @@ DOWNGRADE CANCEL cancels the ongoing downgrade action to cluster.
 ./etcdctl downgrade cancel
 Downgrade cancel success, cluster version 3.5
 ```
+### DIAGNOSIS
+
+`etcdctl diagnosis [flags]` - Collects and analyzes troubleshooting data from a running etcd cluster.
+
+The `diagnosis` command gathers a concise set of diagnostic details from each cluster member by performing several checks, including:
+
+  * **Membership checks**: Verifies the cluster membership information.
+  * **Endpoint status**: Retrieves the status of each endpoint.
+  * **Serializable and linearizable reads**: Performs read operations to validate data consistency.
+  * **Metrics snapshot**: Collects a small snapshot of key metrics.
+
+#### Flags
+
+- `--cluster`: use all endpoints discovered from the cluster member list.
+- `--etcd-storage-quota-bytes`: expected etcd storage quota in bytes (value passed to etcd with `--quota-backend-bytes`).
```

Comment:
For etcd version >= 3.6.0, we are able to get the quota by `endpoint status` query. For versions < 3.6.0, we still need to users to provide a value for this flag.

It's OK to enhance this in a followup PR.